appyters/PrismEXP/templates/form.j2

```diff
@@ -0,0 +1,133 @@
+
+{% block head %}
+{{ super() }}
+{% endblock %}
+
+{% block style %}
+{{ super() }}
+<style>
+
+body {
+  font: 16px Arial;  
+  margin-top: 10%;
+  text-align: center;
+}
+.autocomplete {
+  position: relative;
+  display: inline-block;
+}
+
+input {
+  border: 1px solid transparent;
+  background-color: #f1f1f1;
+  padding: 10px;
+  font-size: 16px;
+}
+
+input[type=text] {
+  background-color: #f1f1f1;
+  width: 100%;
+}
+
+input[type=submit] {
+  background-color: DodgerBlue;
+  color: #fff;
+  cursor: pointer;
+}
+
+.autocomplete-items {
+  position: absolute;
+  border: 1px solid #d4d4d4;
+  border-bottom: none;
+  border-top: none;
+  z-index: 99;
+  /*position the autocomplete items to be the same width as the container:*/
+  top: 100%;
+  left: 0;
+  right: 0;
+}
+
+.autocomplete-items div {
+  padding: 10px;
+  cursor: pointer;
+  background-color: #fff; 
+  border-bottom: 1px solid #d4d4d4; 
+}
+
+.autocomplete-items div:hover {
+  background-color: #e9e9e9; 
+}
+
+.autocomplete-active {
+  background-color: DodgerBlue !important; 
+  color: #ffffff; 
+}
+</style>
+{% endblock %}
+
+{% block body %}
+<div class="container">
+    <div class="mt-3 row">
+        <div class="col-3">
+            <img src="{{ url_for('static', filename='prismexp-icon.png') }}" style="width:180px" class="img-fluid float-right">
+        </div>
+        <div class="col-7 text-left">
+            <h1> PrismEXP </h1>
+          
+    <div> This Appyter calculates predictions using the PrismEXP algorithm. It utilizes precomputed correlation scores and a pretrained model. The correlation scores are computed for 50 gene expression clusters in ARCHS4.
+    <br><br>To start the prediction, choose a gene of interest and provide a GMT file. To provide a GMT file either select one from the Enrichr database or upload a custom file.
+        </div>
+    </div>
+</div>
+{{ super() }}
+{% endblock %}
+
+{% block script %}
+<script>
+require(['jquery'], function ($) {
+  $(document).ready(function () {
+    $('form').submit(function (e) {
+      e.preventDefault();
+      // Replace files with text elements
+      $('form [type="file"]').each(function () {
+        $(this).attr('type', 'text')
+      })
+      // Handle multi-checkbox-fields
+      $('.multi-checkbox-field').each(function (e) {
+        var self = $(this)
+        var target = $(self.attr('data-target'))
+        var value = self.find('input[type="checkbox"]').filter(function () {
+          return this.checked
+        }).map(function () {
+          return this.parentNode.innerText.trim()
+        })
+        target.val(JSON.stringify(Array.from(value)))
+      })
+      // Handle tabs in target fields
+      $('[role="tablist"]').each(function () {
+        var self = $(this)
+        var target = $(this).attr('target')
+        var value = self.find('[role="tab"].active').attr('target')
+        $(target).val(value);
+      })
+      // Submit form
+      this.submit()
+    })
+  })
+})
+</script>
```

Comment:
This script block seems irrelevant given that all of these things are defined in `{{ super() }}`  It was necessary in Drugmonizome_ML Because of tab-field but that is now a part of appyter.